nephio/optional/o2ims/app/deployment.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: o2ims-operator
+  namespace: o2ims
+  labels:
+    app.kubernetes.io/name: nephio-o2ims
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: nephio-o2ims
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: nephio-o2ims
+    spec:
+      securityContext:
+        runAsGroup: 0
+        runAsUser: 0
+      containers:
+      - name: nephio-o2ims
+        image: docker.io/nephio/o2ims-operator:latest
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: 'GIT_REPOSITORY'
```

Comment:
Per @efiacor comment, this may have to change to `UPSTREAM_PKG_REPO` as per the change in utils.py file